lms/djangoapps/courseware/management/commands/dump_course_structure.py

```diff
@@ -95,6 +95,11 @@ def dump_module(module, destination=None, inherited=False, defaults=False):
 
     items = own_metadata(module)
 
+    # HACK: add discussion ids to list of items to export (AN-6696)
+    if module.location.category == 'discussion':
+        if 'discussion_id' not in items:
+            items['discussion_id'] = module.discussion_id
```

Comment:
Perhaps make this even safer by doing something like this:

```python
if 'discussion_id' not in items and hasattr(module, 'discussion_id'):
    items['discussion_id'] = module.discussion_id
```

I'm not sure if it's possible for a discussion module to *not* have a discussion_id attribute.